### ISSUE TITLE
seem to have a compatibility issue with SellJunk

### ISSUE THREAD

**Issue by Acidfrost59** [arkayenro/arkinventory#1160]
Hello Arkayenro,
I hope you are having a great cruise. :)  
I completed the temp fix per #1143 and it worked wonderfully.
However, when I went to a vendor I got the following error with my SellJunk addon: ( I will submit to them, as well).
50x ArkInventory\ArkInventoryStorage.lua:513: attempt to call field 'ObjectLockChanged' (a nil value)
ArkInventory\ArkInventoryStorage.lua:513: in function `?'
...ventory\Libs\CallBackHandler\CallbackHandler-1.0-7.lua:145: in function <...ventory\Libs\CallBackHandler\CallbackHandler-1.0.lua:145>
[string "safecall Dispatcher[3]"]:4: in function <[string "safecall Dispatcher[3]"]:4>
[C]: ?
[string "safecall Dispatcher[3]"]:13: in function `?'
...ventory\Libs\CallBackHandler\CallbackHandler-1.0-7.lua:90: in function `Fire'
...rkInventory\Externals\Ace3\AceEvent\AceEvent-3.0-4.lua:120: in function <...rkInventory\Externals\Ace3\AceEvent\AceEvent-3.0.lua:119>
[C]: ?
SellJunk\SellJunk-3.0.20.lua:99: in function `Sell'
SellJunk\SellJunk-3.0.20.lua:22: in function <SellJunk\SellJunk.lua:22>

They both still work fine, so I am not sure what the deal is, but I suspect it is a Play nice thing.
Best regards,

**Comment by arkayenro** [comment]
odd, the code should be checking for SellJunk and aborting if found.  there shouldnt be a conflict so its probably something else

**Comment by Acidfrost59** [comment]
If it is odd, it is probably me!

**Comment by arkayenro** [comment]
try 3.08.23 and enable junk test mode, it should say that SellJunk is loaded and its aborting

**Comment by Acidfrost59** [comment]
Thank you for your response. I did not have success: I had the same issues
as GreenNite, but I followed your instructions to GreenNite and it is
working beautifully.  I am not sure what the difference would be between
your zip file and Twitch, but Twitch is saying that it is updated to
3.08.23 but the download from Twitch did not work.
Thank you for all you do. I have supported you in the past and will do so
again in the future.
I do not write LUA code but I maintain BPCritterEmotes, and even that small
amount of updating can be difficult at times. I appreciate the work that is
far more in depth.
Best Wishes,
Kris Springer (Acidfrost59)
